### PR TITLE
use asInt(eger) consequenatally

### PR DESCRIPTION
**Pull request by jakob-r**
After discussion in #374

### CODE REVIEW

File: R/renderExampleRunPlotSingleCrit.R [hunk 1]
```diff
@@ -7,7 +7,7 @@ renderExampleRunPlot.MBOExampleRun = function(object, iter, densregion = TRUE,
   requirePackages("ggplot2")
 
   iters.max = object$control$iters
-  assertIntegerish(iter, lower = 0L, upper = iters.max, len = 1L, any.missing = FALSE)
+  iter = asInt(iter, lower = 1L, upper = iters.max)
```
Comment (berndbischl) [inline R/renderExampleRunPlotSingleCrit.R]: the docs say that this is a vec. so asInteger?

File: R/renderExampleRunPlotSingleCrit1d.R [hunk 2]
```diff
@@ -41,7 +41,7 @@ renderExampleRunPlot1d = function(x, iter,
   opt.direction = getMBOInfillCritMultiplier(control$infill.crit)
 
   # if no iterations provided take the total number of iterations in optimization process
-  assertInteger(iter, lower = 0, upper = length(models), len = 1L, any.missing = FALSE)
+  iter = asInt(iter, lower = 0, upper = length(models))
```
Comment (berndbischl) [inline R/renderExampleRunPlotSingleCrit1d.R]: same problem

**Comment by berndbischl** [comment]
rest looks good

File: R/renderExampleRunPlotSingleCrit.R (hunk 1 above)
Comment (jakob-r) [inline R/renderExampleRunPlotSingleCrit.R]: It said len=1 before...

File: R/renderExampleRunPlotSingleCrit1d.R (hunk 2 above)
Comment (jakob-r) [inline R/renderExampleRunPlotSingleCrit1d.R]: same as above

File: R/renderExampleRunPlotSingleCrit.R (hunk 1 above)
Comment (jakob-r) [inline R/renderExampleRunPlotSingleCrit.R]: don't confuse renderExampleRun with renderExampleRunPlot :stuck_out_tongue_winking_eye: 
The last one just generates one Plot

Comment (berndbischl) [inline R/renderExampleRunPlotSingleCrit.R]: sorry, i got confused

**Pull request merged**